index.html

```diff
@@ -35,6 +35,135 @@ <h1>I'm Yohannes</h1>
+                Printing Data </span>
+             <p class="selections"> A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+             <div class="buttons">
+              <button>html</button> 
+               <button>bootstrap</button> 
+              <button>Ruby</button>
+             </div>
+             <p class="invite">See Project</p>
+        </div>
+        <div class="art-item">
+            <span class="data">Professional Art<br>
+                Printing Data </span>
+             <p class="selections"> A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+             <div class="buttons">
+              <button>html</button> 
+               <button>bootstrap</button> 
+              <button>Ruby</button>
+             </div>
+             <p class="invite">See Project</p>
+        </div>
+        <div class="art-item">
+            <span class="data">Professional Art<br>
+                Printing Data </span>
+             <p class="selections"> A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+             <div class="buttons">
+              <button>html</button> 
+               <button>bootstrap</button> 
+              <button>Ruby</button>
+             </div>
+             <p class="invite">See Project</p>
+        </div>
+        
+        </div>
+        </div>
+
+    </section>
+    <section class="about-me">
+        <div class="myresume">
+        <h3 class='h3-title'>About me</h3>
+        <p class="about-paragraph"> Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+        <button class="resume">Get My Resume</button>
+        <hr/>
+        </div>
+        <div class="work-on">
+         <h3 class="best" ><img src="icons/Rectangle 55.png" alt='rectangl'>  Language</h3>
+           <ul class="list">
+               <li>Javascript</li>
+               <li>Ruby</li>
+               <li>Html</li>
+               <li>Css</li>
+           </ul>
+          <h3 class="best" ><img src="icons/Rectangle 56.png" alt="rectangle">  Frameworks</h3>
+             <ul class="list">
+                 <li>Bootstrap</li>
+                 <li>Ruby on Rails</li>
+                 <li>Rspec</li>
+                 <li>CapyBara</li>
+                 <li>Selenium</li>
+             </ul>
+            <h3 class="best" ><img src="icons/Rectangle 57.png"alt='circle'>  Skills</h3>
+               <ul class="list">
+                   <li>Codekit</li>
+                   <li>Github</li>
+                   <li>Codepen</li>
+                   <li>Gitlab</li>
+                   <li>Terminal</li>
+               </ul> 
+            </div>
```

Comment:
- [ ] [Required] You can make all the languages, frameworks, and skills to be enclosed in a `ul` tag. You will then nest the `ul` to house the 'list' just as you have it on lines 157 through 165. Let's try making these changes. It would be fun I guess.
Make sure you style them (font, margin, padding, centered, etc) as it is on Figma:point_down:
![image](https://user-images.githubusercontent.com/45668340/148562846-4cb5d5aa-1bab-4ded-af14-8de02e94ffc8.png)

Reply:
Thanks for the great review you have given me, but for this one, I can't think of a way to do

Reply:
Thanks for the great review you have given me, but for this one, I can't think of a way to do

Reply:
Thanks for the great review you have given me, but for this one, I can't think of a way to do

Reply:
What is the issue exactly? I am here to help in anyway I can

Reply:
Thanks for the great review you have given me, but for this one, I can't think of a way to do

Reply:
Thanks for the great review you have given me, but for this one, I can't think of a way to do

Reply:
Thanks for the great review you have given me, but for this one, I can't think of a way to do

Reply:
honestly, I don't know how to write the Html for this one